app/css/bootstrap.css

```diff
@@ -3260,8 +3258,8 @@ fieldset[disabled] .btn-success.focus {
 .btn-info.active.focus,
 .open > .dropdown-toggle.btn-info.focus {
   color: #fff;
-  background-color: #269abc;
-  border-color: #1b6d85;
+  background-color: #743dfb;
+
```

Comment:
check if this change is making any difference - if not, get it back to how it was